lab-tara/app/src/main/java/com/example/filterablelist/FriendsDetailActivity.java

```diff
@@ -0,0 +1,10 @@
+package com.example.filterablelist;
+
+import android.support.v4.app.Fragment;
+
+public class FriendsDetailActivity extends FriendListActivity {
```

Comment:
This might be better extended from ThemeHeaderFragmentActivity, as it isn't really very related to the List (at least not from a class hierarchy perspective)